app/controllers/api/v1/open_library_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class OpenLibraryController < ApplicationController
+      def show
+        @book = OpenLibraryService.new(params[:isbn]).book_info
+        render json: @book[:result], status: @book[:status?]
+      end
+
```

Comment:
rubocop didn't delete this space? I've seen it in several places

Reply:
the configuration I have of rubocop adds an additional empty line at the end of each class